sync_check_terraform/README.md

```diff
@@ -1,10 +1,6 @@
 # Overview
 
-This folder includes an executable description that facilitates the setup and 
-automatic initiation of the Forest Sync-Check service on a DigitalOcean droplet. 
-The service is designed to perform sync checks on both the calibnet and mainnet networks. 
-Additionally, it sends notifications to the Forest Slack notification channel and 
-the sync check service is configured to restart automatically upon droplet reboot.
+This folder contains an executable description that simplifies the setup and automatic initiation of the Forest Sync-Check service on a DigitalOcean droplet. The service is specifically designed to perform sync checks on both the calibnet and mainnet networks. It also sends notifications to the Forest Slack notification channel. Moreover, the sync check service is configured to automatically restart upon droplet reboot, and the New Relic Infrastructure agent is installed for monitoring purposes.
```

Comment:
nit: "executable description" is kind of a confusing concept. Wouldn't "infrastructure configuration" be more appropriate? In which it would be nice to use that wording across all the readme files.